Cabal/Cabal.cabal

```diff
@@ -459,3 +485,21 @@ test-suite package-tests
   ghc-options: -Wall -rtsopts
   default-extensions: CPP
   default-language: Haskell98
+
+test-suite parser-tests
+  if !flag(parsec-debug)
+    buildable: False
```

Comment:
I'm not sure whether those test would survive in their current form (i.e. comparison to old parser doesn't make sense when there aren't old parser)